client/document/index.jsx

```diff
@@ -312,12 +297,27 @@ class Document extends Component {
 		);
 	}
 }
+function LoadingPlaceholder( { app, sectionName, isWCCOM, showStepContainerV2Loader } ) {
+	const shouldNotShowLoadingLogo =
+		sectionName === 'checkout' || sectionName === 'stepper' || sectionName === 'signup';
 
-function chooseLoadingLogo( { useLoadingEllipsis }, { isWpMobileApp, isWcMobileApp, isWCCOM } ) {
-	if ( useLoadingEllipsis ) {
-		return LoadingEllipsis;
+	if ( shouldNotShowLoadingLogo ) {
+		return showStepContainerV2Loader ? (
+			<Step.Loading />
+		) : (
+			<Loading className="wpcom-loading__boot" />
+		);
 	}
 
+	const LoadingLogo = chooseLoadingLogo( {
+		isWpMobileApp: app?.isWpMobileApp,
+		isWcMobileApp: app?.isWcMobileApp,
+		isWCCOM,
+	} );
+	return <LoadingLogo size={ 72 } className="wpcom-site__logo" />;
```

Comment:
Seems like we still have a few different ways to indicate loading. 

I wish we had just one, following the work this project did:

https://github.com/Automattic/wp-calypso/issues/99256

Also see p58i-kIo-p2

I see you mentioned this is a potential follow-up - I agree it makes sense to have it separately.